src/info.js

```diff
@@ -37,7 +37,7 @@ const getFunctionInfo = async function (srcPath) {
     return {}
   }
 
-  if (extension === '.zip' || extension === '.js') {
+  if (['.js', '.ts', '.zip'].includes(extension)) {
     return { runtime: 'js', name, filename, stat, mainFile, extension, srcPath, srcDir }
   }
```

Comment:
`getMainFile()` should also be fixed to support the new file extensions.

Reply:
As per our chat, I'm temporarily reverting TypeScript support until we're comfortable with how to proceed.